charts/operator/testdata/template-cases.golden.txtar

```diff
@@ -14884,7 +14884,7 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
-        - --configurator-tag=v2.2.4-24.2.5
+        - --configurator-tag=Q8X86Dd4
```

Comment:
are these, which look like generated values going to be problematic?

Reply:
This exact value should not be a problem as many values are generated and are not validated (like tag should be semver'ish).

Reply:
Nope! They're randomly generated strings that are part of the test. In these cases, the override was being specified to such an weird string but the buggy check was checking a value that was actually empty.

If you grep the codebase you'll find that value in `template-cases-generated.txtar`:
```yaml
configurator:
  pullPolicy: IfNotPresent
  repository: R4X
  tag: Q8X86Dd4
```